packages/admin-ui/src/pages/rollups/components/Zkevm.tsx

```diff
@@ -0,0 +1,115 @@
+            {dnpsRequest.data ? (
+              dnpsRequest.data.find((dnp) => dnp.dnpName === zkevmDnpName) ? (
+                <>
+                  {/* zkevmDnp is found */}
+                  {zkevmDnp && zkevmRunning ? (
+                    <Card className={`zkevm-rollup`} shadow={false}>
+                      <div className="avatar">
+                        <img src={zkevmDnp?.avatarUrl || defaultAvatar} alt="avatar" />
+                      </div>
+                      <div className="title">{prettyDnpName(zkevmDnp.dnpName)}</div>
+
+                      <br />
+                      <Alert variant="success">
+                        The Zkevm package is currently <span className="running-text">running</span>
+                      </Alert>
+                      <a
+                        href="http://ui.zkevm-tokens-withdrawal.dappnode/"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        <Button variant="dappnode" className="fullWidthButton">Go to UI</Button>
+                      </a>
+                    </Card>
+                  ) : zkevmDnp && !zkevmRunning ? (
+                    <Card className="not-found">
+                      {/* zkevmDnp is not found */}
+                      <div className="avatar">
+                        <img src={zkevmDnp?.avatarUrl || defaultAvatar} alt="avatar" />
+                      </div>
+                      <div className="title">{zkevmDnp && prettyDnpName(zkevmDnp.dnpName)}</div>
+                      <br />
+                      <Alert variant="warning">
+                        The Zkevm package is installed but not currently running.
+                      </Alert>
+                      <Button
+                        variant="dappnode"
+                        className="fullWidthButton"
+                        onClick={() => navigate(`/packages/my/${zkevmDnpName}/info`)}
+                      >
+                        GO TO PACKAGES
+                      </Button>
+                    </Card>
+                  ) : null}
+                </>
+              ) : (
+                <Card className="not-found">
+                  {/* zkevmDnp is not found */}
+                  <div className="avatar">
+                    <img src={zkevmDnp?.avatarUrl || defaultAvatar} alt="avatar" />
+                  </div>
+                  <div className="title">{zkevmDnp && prettyDnpName(zkevmDnp.dnpName)}</div>
+                  <br />
+                  <p>
+                    You must install the Zkevm package
+                  </p>
+                  <Button
+                    variant="dappnode"
+                    className="fullWidthButton"
+                    onClick={() =>
+                      navigate(`${getInstallerPath(zkevmDnpName)}/${zkevmDnpName}`)
+                    }>GET</Button>
+                </Card>
+              )
+            ) : null}
```

Comment:
Isnt better to embed the row + column inside the conditional? instead of having a possible Row + column with an empty child component